vocab_files/attribute_concepts/type-of-soil-observation.ttl

```diff
@@ -9,7 +9,7 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
     a skos:Concept ;
     dcterms:source "Ecological Field Monitoring protocols - Soil module , draft v0.1, 30/11/2021" ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
-    skos:definition "The soil profile may be described using several types of observation. E.g. is 'Soil pit'." ;
+    skos:definition "The soil profile may be described using several types of observation. E.g. is 'Soils samples obtained via the digging of a pit to expose the soil profile'." ;
```

Comment:
Ah so it's the type of soil observation. I think this is straight out of the YB.

In that case, you are right, soil pit is one of the soil type observations, but not sure why it is mentioning soil profile?

Maybe it should just be:
> The soil ~profile~ may be described using several types of observation. E.g. ~is~ 'Soil pit'.

Reply:
yeah this make sense, I'll update as so